src/main/java/tech/stackable/hadoop/StackableAccessControlEnforcer.java

```diff
@@ -197,7 +170,21 @@ public void checkPermission(
   @Override
   public void checkPermissionWithContext(INodeAttributeProvider.AuthorizationContext authzContext)
       throws AccessControlException {
-    OpaAllowQuery query = new OpaAllowQuery(new OpaAllowQuery.OpaAllowQueryInput(authzContext));
+    // When executing "hdfs dfs -du /" the path is set to null. This does not worsen security, as
+    // "/" is the highest level of access that a user can have.
+    if (authzContext.getOperationName().equals("contentSummary")
+        && authzContext.getPath() == null) {
+      authzContext.setPath("/");
+    }
+
+    final Object query;
```

Comment:
```suggestion
    Object query;
```

We don't use final elsewhere